src/components/ContentNode/TabNavigator.vue

```diff
@@ -74,7 +74,7 @@ export default {
 @import 'docc-render/styles/_core.scss';
 
 .TabNavigator {
-  @include space-out-between-siblings($stacked-margin-xlarge);
+  @include space-out-between-siblings(var(--stacked-margin-xlarge));
```

Comment:
(Same comment about mixing Sass mixins/functions with CSS vars here)

Reply:
I think it should be fine, as it should get passed as a string kind of :D